app/validators/region_validator.rb

```diff
@@ -0,0 +1,43 @@
+class RegionValidator < ActiveModel::Validator
+  PROVINCES = %w[AB BC MB NB NL NS ON PE QC SK NT NU YT].freeze
+
+  def validate(record)
+    country = ISO3166::Country.find_country_by_alpha3(record.country)
+
+    unless country
```

Comment:
`if country.blank?` ?

Reply:
`find_country_by_alpha3` returns either a `ISO3166::Country` or `nil`, so I think `blank?` is unnecessary (and because of that a little misleading). I'd be happy with `if country.nil?`, if that would be preferable?

Reply:
yeah I go with if over unless when possible. I also like to be explicit with what you’re checking for so nil? would work.

Reply:
Works for me! Swapped out for `nil?` in 02e3367